---
pr2_collision_monitor/FingerState state
